pandas/core/frame.py

```diff
@@ -3983,7 +3983,7 @@ def drop(
         errors="raise",
     ):
         """
-        Drop specified labels from rows or columns.
+        Drop specified labels from rows or columns and return a new DataFrame object.
```

Comment:
I think you can revert this line as it contradicts ``inplace=True``. The update you have a few lines below should be sufficient

Reply:
Thanks for the review! Changed it in commit 15bd288974e0007459990879f3ce656c4884dd73